tools/pubsys/src/aws/validate_ssm/mod.rs

```diff
@@ -80,52 +80,56 @@ pub async fn validate(
 
     // Retrieve the SSM parameters using the SsmClients
     info!("Retrieving SSM parameters");
-    let parameters = get_parameters_by_prefix(&ssm_clients, ssm_prefix).await;
+    let parameters = get_parameters_by_prefix(&ssm_clients, ssm_prefix)
+        .await
+        .into_iter()
+        .map(|(region, result)| {
+            (
+                region,
+                match result {
+                    Ok(result) => Ok(result),
+                    Err(_) => Err(error::Error::UnreachableRegion {
+                        region: region.to_string(),
+                    }),
```

Comment:
Does it make sense for this to hold onto the original error?

I think we just want to make sure the reason for the unavailability can be surfaced. Ensuring that it is logged somewhere at `ERROR` level probably suffices.